source/cloud-native-platform/path-to-live/public-dns.html.md.erb

```diff
@@ -39,9 +39,22 @@ Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull
   record: "{copy endpoint hostname from frontdoor}"
 ```
 
-Where:
 
-`${env}`  =  Name of the environment for which you are creating the recordsets. Valid values are `sbox` or `prod`.
+## Verify custom domain on the Frontdoor
+- Create TXT records in the appropriate zone file for environments.
+Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1409/files),
```

Comment:
you need to add the frontdoor config first to get the DNS records? 

I think the order should be:

1. FrontDoor config
2. DNS TXT records
3. CNAME / Alias records

should be 3 PRs (bit of a pain unfortunately)

Reply:
think it should be, I have updated my PR now
1. CNAME / alias records
2. Frontdoor Config